go get -u github.com/reviewdog/reviewdog/cmd/reviewdog failing

error:
```
$ go get -u github.com/reviewdog/reviewdog/cmd/reviewdog
01:49
115 # github.com/reviewdog/reviewdog/doghouse/server
116 /go/src/github.com/reviewdog/reviewdog/doghouse/server/github.go:42:67: cannot use opt.IntegrationID (type int) as type int64 in argument to ghinstallation.NewAppsTransport
117 /go/src/github.com/reviewdog/reviewdog/doghouse/server/github.go:48:53: cannot use opt.IntegrationID (type int) as type int64 in argument to ghinstallation.New
118 /go/src/github.com/reviewdog/reviewdog/doghouse/server/github.go:48:72: cannot use int(installationID) (type int) as type int64 in argument to ghinstallation.New
122 ERROR: Job failed: exit code 1
```

golang 1.9.1

--- a/doghouse/server/github.go
+++ b/doghouse/server/github.go
@@ -39,13 +39,13 @@ func NewGitHubClient(ctx context.Context, opt *NewGitHubClientOption) (*github.C
 
 func githubAppTransport(ctx context.Context, client *http.Client, opt *NewGitHubClientOption) (http.RoundTripper, error) {
 	if opt.RepoOwner == "" {
-		return ghinstallation.NewAppsTransport(getTransport(client), opt.IntegrationID, opt.PrivateKey)
+		return ghinstallation.NewAppsTransport(getTransport(client), int64(opt.IntegrationID), opt.PrivateKey)
 	}
 	installationID, err := findInstallationID(ctx, opt)
 	if err != nil {
 		return nil, err
 	}
-	return ghinstallation.New(getTransport(client), opt.IntegrationID, int(installationID), opt.PrivateKey)
+	return ghinstallation.New(getTransport(client), int64(opt.IntegrationID), installationID, opt.PrivateKey)
 }
 
 func getTransport(client *http.Client) http.RoundTripper {

--- a/go.mod
+++ b/go.mod
@@ -6,8 +6,7 @@ require (
 	cloud.google.com/go v0.46.3
 	cloud.google.com/go/datastore v1.0.0
 	cloud.google.com/go/storage v1.0.0 // indirect
-	github.com/bradleyfalzon/ghinstallation v0.1.2
-	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/bradleyfalzon/ghinstallation v1.0.0
 	github.com/google/go-cmp v0.3.1
 	github.com/google/go-github/v28 v28.1.1
 	github.com/haya14busa/secretbox v0.0.0-20180525171038-07c7ecf409f5

